src/Kernel32.Desktop/Kernel32+CreateFileFlags.cs

```diff
@@ -0,0 +1,167 @@
+            /// </summary>
+            SessionAwareFlag = 0x00800000,
+
+            /// <summary>
+            /// Access is intended to be sequential from beginning to end. The system can use this as a hint to optimize file caching.
+            /// This flag should not be used if read-behind (that is, reverse scans) will be used.
+            /// This flag has no effect if the file system does not support cached I/O and <see cref="NoBufferingFlag"/>.
+            /// For more information, see the Caching Behavior section of this topic.
+            /// </summary>
+            SequentialScanFlag = 0x08000000,
+
+            /// <summary>
+            /// Write operations will not go through any intermediate cache, they will go directly to disk.
+            /// For additional information, see the Caching Behavior section of this topic.
+            /// </summary>
+            WriteThroughFlag = 0x80000000,
+
+            /// <summary>
+            /// When this flag is present, one or more other flags ending with Security may also be specified.
+            /// </summary>
+            SqosPresentSecurity = 0x00100000,
+
+            /// <summary>
+            /// Impersonates a client at the Anonymous impersonation level.
+            /// </summary>
+            AnonymousSecurity = SecurityImpersonationLevel.SecurityAnonymous << 16,
+
+            /// <summary>
+            /// The security tracking mode is dynamic. If this flag is not specified, the security tracking mode is static.
+            /// </summary>
+            ContextTrackingSecurity = 0x00040000,
+
+            /// <summary>
+            /// Impersonates a client at the Delegation impersonation level.
+            /// </summary>
+            DelegationSecurity = SecurityImpersonationLevel.SecurityDelegation << 16,
+
+            /// <summary>
+            /// Only the enabled aspects of the client's security context are available to the server. If you do not specify this flag, all aspects of the client's security context are available.
+            /// This allows the client to limit the groups and privileges that a server can use while impersonating the client.
+            /// </summary>
+            EffectiveOnlySecurity = 0x00080000,
+
+            /// <summary>
+            /// Impersonates a client at the Identification impersonation level.
+            /// </summary>
+            IdentificationSecurity = SecurityImpersonationLevel.SecurityIdentification << 16,
+
+            /// <summary>
+            /// Impersonate a client at the impersonation level. This is the default behavior if no other flags are specified along with the SECURITY_SQOS_PRESENT flag.
+            /// </summary>
+            ImpersonationSecurity = SecurityImpersonationLevel.SecurityImpersonation << 16,
+        }
+    }
+}
```

Comment:
How easy would it be to include a newline character at the end of the file to keep some tools happy (and remove the red warning indicator GitHub adds)?